src/main/java/com/opencart/pages/HeaderPage.java

```diff
@@ -28,17 +28,40 @@
     @FindBy(css = "* button > i.fa.fa-search")
     private WebElement searchButton;
 
-    @FindBy(css = "i[class=\"fa fa-heart\"] + .hidden-xs.hidden-sm.hidden-md")
-    private WebElement wishListButton;
+    @FindBy (id = "cart-total")
+    private WebElement shoppingCartButton;
 
-    @FindBy(css = "button[onclick=\"wishlist.add('43');\"]")
-    private WebElement macbookWishListButton;
+    @FindBy (css = ".table.table-bordered tr:nth-child(3) .text-right:last-child")
+    private WebElement productVatTaxRate;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(3) .text-right:first-child strong")
+    private WebElement productVatTaxName;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(2) .text-right:last-child")
+    private WebElement productEcoTaxRate;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(2) .text-right:first-child strong")
+    private WebElement productEcoTaxName;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(1) .text-right:last-child")
+    private WebElement productPrice;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(4) .text-right:last-child")
+    private WebElement productTotalPrice;
```

Comment:
Why did you locate all these elements into HeaderPage? I think it relates to ProductContainer

Reply:
This is WebElements from shopping cart from header button "ShoppingCart" (see screenshot).
I thought ProductContainer is that field with photo, Title, Description, Buttons: WishList, Shopping Cart, etc.. (see screenshot). 
Is it better to put the mentioned WebElements to ProductContainer?

<img width="1036" alt="Screen Shot 2020-11-30 at 11 22 24" src="https://user-images.githubusercontent.com/73054470/100591473-7f138600-32fe-11eb-9b65-18fc70fef09e.png">

<img width="1040" alt="Screen Shot 2020-11-30 at 11 21 17" src="https://user-images.githubusercontent.com/73054470/100591327-41aef880-32fe-11eb-9584-f819ae3dcdeb.png">

Reply:
Oh, yeah, you are right, I thought it is related to product characteristics

Reply:
But before my approval, please resolve conflicts for safely merge

Reply:
resolved